CHANGES.md

```diff
@@ -1,5 +1,191 @@
+    in the outbound proxy
+  * Added support for dynamic request routing based on HTTPRoutes
+  * Added HTTP circuit breaking
+  * Added `outbound_route_backend_http_requests_total`,
+    `outbound_route_backend_grpc_requests_total`, and
+    `outbound_http_balancer_endpoints` metrics
+  * Changed the proxy's behavior when traffic splitting so that only services
+    that are not in failfast are used. This will enable the proxy to manage
+    failover without external coordination
+  * Updated tokio (async runtime) in the proxy which should reduce CPU usage,
+    especially for proxy's pod local (i.e in the same network namespace)
+    communication
+
+* Policy Controller
+  * The policy controller now discovers outbound policy configurations from
+    HTTPRoutes that target Services.
+  * Added OutboundPolicies API, for use by `linkerd-proxy` to route
+    outbound traffic
+  * Added Prometheus `/metrics` endpoint to the admin server, with process
+    metrics
+  * Fixed QueryParamMatch parsing for HTTPRoutes
+  * Added the policy status controller which writes the `status` field to
+    HTTPRoutes when a parent reference Server accepts or rejects it
+
+* linkerd-proxy-init
+  * Changed `proxy-init` iptables rules to be idempotent upon init pod
+    restart (thanks @jim-minter!)
+  * Improved logging in `proxy-init` and `linkerd-cni`
+  * Added a `proxyInit.privileged` setting to control whether the `proxy-init`
+    initContainer runs as a privileged process
+
+* CNI
+  * Added static and dynamic port overrides for CNI eBPF to work with socket-level
+    load balancing
+  * Updated `network-validator` helm charts to use `proxy-init` resources
+
+* Viz
+  * Added `tap.ignoredHeaders` Helm value to the linkerd-viz chart. This value
+    allows users to specify a comma-separated list of header names which will be
+    ignored by Linkerd Tap (thanks @ryanhristovski!)
+  * Removed duplicate SecurityContext in Prometheus manifest
+  * Added new flag `--viz-namespace` which avoids requiring permissions for
+    listing all namespaces in `linkerd viz` subcommands (thanks @danibaeyens!)
+  * Removed the TrafficSplit page from the Linkerd viz dashboard (thanks
+    @h-dav!)
+  * Introduced new values in the `viz` chart to allow for arbitrary annotations
+    on the `Service` objects (thanks @sgrzemski!)
+  * Added an optional AuthorizationPolicy to authorize Grafana to Prometheus
+    in the Viz extension
+
+* Multicluster
+  * Removed duplicate AuthorizationPolicy for probes from the multicluster
+    gateway Helm chart
+  * Updated wording for linkerd-multicluster cluster when it fails to probe a
+    remote gateway mirror
+  * Added multicluster gateway `nodeSelector` and `tolerations` helm parameters
+  * Added new configuration options for the multicluster gateway:
+    * `gateway.deploymentAnnotations`
+    * `gateway.terminationGracePeriodSeconds` (thanks @bunnybilou!)
+    * `gateway.loadBalancerSourceRanges` (thanks @Tyrion85!)
+
+* Helm
```

Comment:
these are changes to the control plane templates/manifests.  I'd just call this section "Control Plane"

Reply:
okay, would you merge that into the "policy controller" and "destination controller" sections, or have three separate ones?:thumbsu

Reply:
or merge them all together in a Control Plane section.  I don't have a preference, but I don't think "Helm" is the right heading.

Reply:
I'd vote for "Control Plane" personally, yeah.

Reply:
@adleong would you put _all_ the changes in this subsection in the Control Plane section?

Reply:
okay done in 5f698f2a33520fbec1b08155f956370dc5eabe84